clap_derive/src/derives/args.rs

```diff
@@ -109,6 +109,7 @@ pub fn gen_for_struct(
             clippy::nursery,
             clippy::cargo,
             clippy::suspicious_else_formatting,
+            clippy::almost_swapped,
```

Comment:
Can you also do this for
- https://github.com/clap-rs/clap/blob/master/clap_derive/src/derives/into_app.rs
- https://github.com/clap-rs/clap/blob/master/clap_derive/src/derives/value_enum.rs

Reply:
done 🫡